src/libs/common/include/storm_platform.h

```diff
@@ -0,0 +1,88 @@
+#pragma once
+#ifdef _WIN32
+inline const char *convert_path_sep(const char *cPath)
+{
+    return cPath;
+}
+#else
+
+#include <limits.h>
+
+#include "storm_d3dx9.h"
+#include "wine/winuser.rh"
+#include "safe_str_lib.h"
+#undef EXTERN // fix for token.h:72:5: error: expected identifier EXTERN,
+
+#define MAX_PATH PATH_MAX
+#define _MAX_FNAME NAME_MAX
+#define MAKELONG(low, high) ((int32_t)(((uint16_t)(low)) | (((uint32_t)((uint16_t)(high))) << 16)))
+
+inline char *convert_path_sep(const char *cPath)
```

Comment:
don't make those inline. instead, put the implementation into `common/src`. in that way you won't need to bloat compilation with headers and doubtful defines, moving them into cpp file

`#undef` (especially multiple) is also not a good idea

Reply:
Done. I hope that I fixed it correctly.